apache/superset
# feat(dashboard): Implement empty states for dashboard

### SUMMARY
This PR implements chart empty states for 3 scenarios:

1. Dashboard has no components added
2. Dashboard is in edit mode and has no components added yet
3. Native filters bar is empty

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/15073128/153859190-ebc9d41a-d4dd-4796-9572-0e4dc1d8dd40.png">

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/15073128/153860494-6252c786-1bd8-4f60-8059-a29620b069a6.png">

### TESTING INSTRUCTIONS
1. Create a dashboard
2. Verify that there is an empty state in native filters bar when there are no native filters added
3. Verify that there is an empty state in the middle of the dashboard when there are no components added
4. Go to edit mode and verify that there is an empty state when there are no components added
5. Click the button "Create a new chart" button in edit mode's empty state and verify that it redirects you to chart creator

### ADDITIONAL INFORMATION



CC @kasiazjc

https://app.shortcut.com/preset/story/37784/dashboard-empty-states
